transport/src/androidUnitTest/kotlin/transport/core/StateMachineTest.kt

```diff
@@ -287,7 +287,7 @@ class StateMachineTest {
     }
 
     @Test
```

Comment:
Just wondering why this test doesn't follow the same naming structure of "when..."